src/WebClient.spec.js

```diff
@@ -500,15 +501,95 @@ describe('WebClient', function () {
   });
 
   describe('has rate limit handling', function () {
-    it('should expose retry headers in the response');
+    it('should expose retry headers in the response', function() {
+      const scope = nock('https://slack.com')
+        .post(/api/)
+        .reply(200, { ok: true }, {
```

Comment:
hmmm... not quite. we would only ever see `retry-after` when the response has a status code of 429. i'm not 100% sure what the body would contain, but `{ ok: true }` seems wrong to me, too.

Reply:
my bad; i was supposed to change that to 429, but forgot about it. Thanks!